new-frontiers-14/frontier-station-14
# Fixed contraband parenting on some weapons and clothes

## About the PR
- Due to parenting order weapons obtained through expeditions were not marked as incorrect contraband type. This PR fixes this issue.
- Added new C3 tier reserved for expedition weapons that should be considered illegal to own.
- Distributed contraband definitions to a number of items: frontier specific weapons (argenti, novalite, turbolaser etc.), turrets, some clothing articles.
- Changed mk58 to C1 to match weapon's description.
- Classified certain craftable clown shoes as unredeemable C3 contraband for lols.

## Why / Balance
Clarity

## How to test
1. Spawn liberation station vendomat, buy weapons from it, inspect them - should be C1.
2. Go through the list of weapons in `\Resources\Prototypes\_NF\Entities\Objects\Weapons\Guns\expedition_guns.yml` using `spawn` console command. Items in that list should be either C2 or C3.
3. Warp to NFSD outpost, navigate to armory and inspect turrets on turret locker. Those should be considered as restricted to NFSD/Command. Deploy the turrets, inspect them again, should be the same.
4. Dive on couple expeditions and  inspect weapons spawned there, should be C2 or C3.

## Media

## Requirements

## Breaking changes
Removed `UnregisteredFirearmBase`

**Changelog**
:cl: erhardsteinhauer
- tweak: Contraband status of weapons sourced from expeditions should become more apparent upon inspection.
- fix: Classified weapons available from LiberationStation as C1.
